src/test/integration/oauth-test.js

```diff
@@ -103,11 +103,23 @@ describe('oauth/', function() {
         secret: CLIENT_SECRET,
       },
       auth: {
-        tokenHost: `https://127.0.0.1:${server.address().port
-        }${Constants.OAUTH_PATH}`,
+        tokenHost: `https://127.0.0.1:${server.address().port}`,
+      },
+      options: {
+        authorizationMethod: 'body',
+      },
+      http: {
+        headers: {
+        },
       },
     }, configProvided || {});
 
+    const clientId = encodeURIComponent(config.client.id).replace(/%20/g, '+');
```

Comment:
I'm pretty sure this isn't necessary since encodeURIComponent mostly exists as a way of allowing client ids and secrets to contain colons (if I remember correctly)

Reply:
See other comment. This was just done to match watch that library does.